pkgs/by-name/gp/gpick/package.nix

```diff
@@ -2,11 +2,10 @@
   stdenv,
   fetchFromGitHub,
   cmake,
-  wrapGAppsHook3,
   boost,
-  pkg-config,
-  gtk3,
   ragel,
+  pkg-config,
+  wrapGAppsHook3,
```

Comment:
Why move `pkg-config` and `wrapGAppsHook3` around? This could just be deleting `gtk3`.